src/index.js

```diff
@@ -18,6 +19,7 @@ let reduxStore = {
     risks: risks,
     asn: asn,
     cubeByRiskByCountry: {},
+    cubeByRiskByASN: {},
     cubeByRiskByAS: tempData,
```

Comment:
`cubeByRiskByASN: {}` is redux store attribute for ASPage data.
`cubeByRiskByAS: tempData` is temporary here, I am just copying `cubeByRiskByCountry: {}` but with top ~9 AS data. Once our API is ready, it will be removed. I agree attribute naming is not clear here..